src/styles/components/_watch.scss

```diff
@@ -0,0 +1,49 @@
+.watch {
+  position: relative;
+  margin: 30px auto;
+  width: $size-watch;
+  height: $size-watch;
+  border: 7px dotted #000;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    background-color: orange;
+    border-radius: 50%;
+    width: 50px;
+    height: 50px;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  &__seconds {
+    position: absolute;
+    background-color: green;
+    width: 10px;
+    height: $size-watch / 2 - 2vmin;
+    top: 25%;
+    left: 50%;
+
+    transform-origin: 50% 100%;
+    animation-name: rotate;
+    animation-duration: 60s;
+    animation-timing-function: $seconds-timing-function;
+    animation-iteration-count: infinite;
+  }
+
+  &__minuts {
+    position: absolute;
+    background-color: blue;
+    width: 15px;
+    height: $size-watch / 4;
+    top: 37%;
+    left: 50%;
+
+    transform-origin: 50% 100%;
+    animation-name: rotate;
+    animation-duration: 3600s;
+    animation-timing-function: $minutes-timing-function;
+    animation-iteration-count: infinite;
```

Comment:
add one more class with shared properties for both arrows to avoid duplication

Reply:
done